ui/src/criteria/attribute.tsx

```diff
@@ -18,17 +24,36 @@ interface Config extends CriteriaConfig {
 
 interface Data extends Config {
   selected: Selection[];
+  min: number | undefined;
+  max: number | undefined;
 }
 
 type AttributeEditProps = {
   dispatchFn: (data: Data) => void;
   data: Data;
 };
 
-@registerCriteriaPlugin("attribute", (config: CriteriaConfig) => ({
-  ...(config.plugin as Config),
-  selected: [],
-}))
+@registerCriteriaPlugin(
+  "attribute",
+  (underlay: Underlay, config: CriteriaConfig) => {
+    const data = config.plugin as Config;
+    const attributeFilterHint = underlay.entities
+      .find((g) => g.name === underlay.primaryEntity)
+      ?.attributes?.find(
+        (attribute) => attribute.name === data.attribute
+      )?.attributeFilterHint;
+    const integerBoundsHint = attributeFilterHint?.integerBoundsHint;
+    const minBound = integerBoundsHint?.min;
```

Comment:
nit: I don't know that these variables are necessary, might just inline this below instead.